webpack.common.js

```diff
@@ -233,6 +233,10 @@ const config = [
 			'eventespresso-core-blocks': [
 				assets + 'blocks/index.js',
 			],
+			/** implement once there's javascript to output
+			'eventespresso-core-blocks-frontend': [
+				assets + 'blocks/index-frontend.js',
+			],*/
```

Comment:
Placeholder, we don't actually need to build this yet because there is nothing to build, but when we do we can just comment this out.  I debated whether I should do this or not because in general I dislike adding placeholders like this, but in this case I think its of benefit for our future selves to have.